core/domain/platform_parameters.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
```

Comment:
I'm not sure about this file, why we need to have dummy platform parameters? If there is some reason I would still place the to the platform_parameter_registry.py instead.

Reply:
This file is used for the definition of parameters (and feature flags) as per @BenHenning 's comment, as they there might be tens of parameters and the file can be hundreds of lines long, it will be better to separate the definition and the domian object classes and `Registry`.
For the dummy parameters, they are created for two purposes:
* they indicate the usage of `platform_parameters.py` and how parameters & feature flags should be created.
* they will be used in the incoming e2e test that goes through the flow of feature gating.

Reply:
@vojtechjelinek we're likely going to have dozens of parameters in the long-term, and we will never be able to remove them if they have any Android dependencies. It's confusing to conflate a list of parameters & the registry (I don't like how we do this for the existing config values, either). It seems much cleaner to have a single file responsible for defining this list. In the future, we may even want to further split this up if this file becomes too long.